src/components/quotes.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=computers', {
+        headers: {
+          'X-Api-Key': '0oWa33wHCcpfBcURu9UPv39A2rSCT1DcyejUtSf8',
+          'Content-Type': 'application/json',
+        },
+      });
+
+      if (response.ok) {
+        const data = await response.json();
+        const randomIndex = Math.floor(Math.random() * data.length);
+        setQuote(data[randomIndex]?.quote);
+        setIsLoading(false);
+      } else {
+        setError('Error fetching quotes');
```

Comment:
- [ ] Your error handling does not work as expected. In checking for error handling, I disconnected the internet, with the expectation that the quote is not fetched, so should display an error. It only keeps showing me `Loading...` without ever rendering the error message. Kindly ensure that the error message is displayed on the screen when the quote cannot be fetched.